### PR TITLE
[DX-1438] Update installation.md

### PR DESCRIPTION
**Pull request by letzya**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link
https://deploy-preview-4827--tyk-docs.netlify.app/docs/nightly/getting-started/installation/

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

___

### **PR Type**
Documentation

___

### **Description**
- Updated the title and description of the installation guide for better clarity and SEO.
- Added a welcome message to introduce the guide.
- Structured the guide into sections for different user levels: Newbie, Advanced Beginners and up.
- Provided detailed descriptions and links for different Tyk installation options: Tyk Self-Managed, Tyk Cloud, Tyk Hybrid, and Open Source Tyk Gateway.
- Included a section for users who are unsure about which installation to choose, with a link to a comparison page and contact information.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>installation.md</strong><dd><code>Revise and expand Tyk installation guide for clarity and detail</code></dd></summary>
<hr>

tyk-docs/content/getting-started/installation.md
<li>Updated the title and description for clarity.<br> <li> Added a welcome message and structured the guide into sections for <br>different user levels.<br> <li> Provided detailed descriptions and links for Tyk Self-Managed, Tyk <br>Cloud, Tyk Hybrid, and Open Source Tyk Gateway.<br> <li> Added a section for users unsure about which installation to choose, <br>with a link to a comparison page and contact information.<br>

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4827/files#diff-043dc73b8e163ac93126032f7000ea4caad7b63399c85b5c0932c3ebd2f95a3f">+32/-13</a>&nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: tyk-docs/content/getting-started/installation.md [hunk 1]
```diff
@@ -1,21 +1,56 @@
...
 date: 2017-03-08T18:15:30+13:00
-title: Install
-tags: ["Tyk Installation Options", "Tyk Open Source API Gateway", "Tyk Self-Managed Installation", "Tyk Cloud Installation"]
-description: "Choosing the Tyk installation that suits your needs"
-menu: 
-    main:
-        parent: Getting Started
-weight: 2
+title: Guide to tyk installations
+tags: ["Tyk Installation Options", “Guide to Tyk installations”, "Tyk Open Source API Gateway", "Tyk Self-Managed Installation", "Tyk Cloud Installation"]
```
Comment (letzya) [inline tyk-docs/content/getting-started/installation.md]:
```suggestion
tags: ["installing", "installation", "Tyk Installation Options", “Guide to Tyk installations”, "Tyk Open Source API Gateway", "Tyk Self-Managed Installation", "Tyk Cloud Installation"]
```

File: tyk-docs/content/getting-started/installation.md [hunk 2]
```diff
@@ -1,21 +1,56 @@
 --- 
 date: 2017-03-08T18:15:30+13:00
-title: Install
-tags: ["Tyk Installation Options", "Tyk Open Source API Gateway", "Tyk Self-Managed Installation", "Tyk Cloud Installation"]
-description: "Choosing the Tyk installation that suits your needs"
-menu: 
-    main:
-        parent: Getting Started
-weight: 2
+title: Guide to tyk installations
```
Comment (letzya) [inline tyk-docs/content/getting-started/installation.md]:
```suggestion
title: Guide to Tyk Installations
```

File: tyk-docs/content/getting-started/installation.md [hunk 3]
```diff
@@ -1,21 +1,56 @@
...
-weight: 2
+title: Guide to Tyk Installations
+tags: ["installing", "installation", "Tyk Installation Options", “Guide to Tyk installations”, "Tyk Open Source API Gateway", "Tyk Self-Managed Installation", "Tyk Cloud Installation"]
+description: "This guide helps you choose the Tyk installation that best suits your needs."
 aliases:
     - /try-out-tyk/tutorials/tutorials/
 ---
 
-Please head on over to the solution you want to setup:
+Welcome to the Tyk Installation Guide! Whether you're new to Tyk or an experienced user, this guide will help you find the installation option that best meets your needs. Let's get started!
```
Comment (letzya) [inline tyk-docs/content/getting-started/installation.md]:
```suggestion
Welcome to the Tyk Installation Guide! Whether you're new to Tyk or an experienced user, this guide will help you find the installation model that best meets your needs. Let's get started!
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
PR LGTM! @letzya

I made some minor changes to add relative paths to button left etc.

What is the URL for the comparison page?

The current link included below gives a 404
[Comparison Page](https://tyk.io/about/contact)

File: tyk-docs/content/getting-started/installation.md [hunk 4]
```diff
@@ -1,21 +1,82 @@
...
+
+<br/>
+
+{{< button_left href="apim/open-source/installation" color="green" content="Install Tyk OSS" >}}
+
+---
+
+## Not Sure Which to Choose?
+
+Visit our [Comparison Page](https://tyk.io/about/contact) to understand the different installation types and their benefits.
```
Comment (letzya) [inline tyk-docs/content/getting-started/installation.md]:
```suggestion
Visit our [Comparison Page]({{< ref "/apim" >}}) to understand the different installation types and their benefits.
```

**Comment by letzya** [comment]
> PR LGTM! @letzya
> 
> I made some minor changes to add relative paths to button left etc.
> 
> What is the URL for the comparison page?
> 
> The current link included below gives a 404 [Comparison Page](https://tyk.io/about/contact)

Thanks!!!
I messed up with it. The comparison is from the current page, not my addition, I just messed up while copy/pasting. it's /apim

**Review by dcs3spp — APPROVED** [review]

**Comment by dcs3spp** [comment]
@sedkis Would you be able to peer review?

**Review by sedkis — COMMENTED** [review]
LGTM!

**Pull request merged**